### DISCUSSION TITLE
Python 3.13 wheels

### DISCUSSION THREAD

**Discussion by mgineer85** [jim-easterbrook/python-gphoto2#178] [General]
Hey @jim-easterbrook ,

for automated testing on github actions, it would be great if you could add python 3.13 wheels to the builds as it exists for 3.12 already.
You think it's possible without much effort?

**Comment by jim-easterbrook** [comment]
The current version of SWIG doesn't officially support Python 3.13, but a new release (which will) is expected this month.

**Comment by jim-easterbrook** [comment]
Despite SWIG not supporting Python 3.13 everything python-gphoto2 uses seems to be OK. I've just released version 2.5.1 which includes binary wheels for Python up to 3.13 and also arm64 macOS.

**Reply by mgineer85** [reply]
thank you very much! Actions seem to work fine again :)
